trackpy/tests/test_predict.py

```diff
@@ -212,13 +248,159 @@ def setUp(self):
             predict.instrumented()(predict.ChannelPredict),
             3, 'y', minsamples=3)
         self.mkframe = self._channel_frame
+
     def _channel_frame(self, n=1, Nside=3):
-        xg, yg = np.mgrid[:Nside,:Nside]
+        xg, yg = np.mgrid[:Nside,:Nside] * 100
         dx = 0.
-        dy = (n - 1) * np.sqrt(2)
+        dy = (n - 100) * np.sqrt(2)
         return pandas.DataFrame(
             dict(x=xg.flatten() + dx, y=yg.flatten() + dy, frame=n))
 
+
+## find_link prediction tests
+# Define a mixin that converts a normal prediction test class into one
+# that uses find_link.
+
+class LinkNewPredictTest(object):
+    def get_linker_iter(self, pred):
+        return functools.partial(pred.wrap, trackpy.link_simple_df_iter)
+
+    def get_linker(self, pred):
+        return functools.partial(pred.wrap_single, trackpy.link_simple_df_iter)
+
+    def get_unwrapped_linker(self):
+        return trackpy.link_simple_df_iter
+
+
+class NewBaselinePredictTests(LinkNewPredictTest, BaselinePredictTests):
+    pass
+
+
+class NewNearestVelocityPredictTests(LinkNewPredictTest, NearestVelocityPredictTests):
+    pass
+
+
+class NewDriftPredictTests(LinkNewPredictTest, DriftPredictTests):
+    pass
+
+
+class NewChannelPredictXTests(LinkNewPredictTest, ChannelPredictXTests):
+    pass
+
+
+class NewChannelPredictYTests(LinkNewPredictTest, ChannelPredictYTests):
+    pass
+
+
+class FindLinkPredictTest(object):
+    def setUp(self):
+        super(FindLinkPredictTest, self).setUp()
+        self.linker_opts = dict(separation=10, diameter=15)
+        # Disable certain tests that are redundant here
+        # and would require more code to support.
+        self.coords_via_images = True
```

Comment:
This approach would allow you to skip the currently-failing `test_quadrature_distances` test for find_link